docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -0,0 +1,176 @@
+
+== Trying Out the DMT
+
+The instructions below for trying out the DMT use a map in Hazelcast 4.x.y cluster whose data is migrated to that of Hazelcast {full-version} cluster.
+
+The steps can be summarized as follows:
+
+. Start a source cluster.
+. Create a map and put data into it.
+. Start the target cluster.
+. Start the migration cluster.
+. Migrate map data from the source cluster to the target.
+. Verify the data in the target cluster.
+
+=== Step 1. Start the Source Cluster
+
+Run the following Docker command to start a one-member cluster.
+
+[source,shell]
+----
+$ docker run -p 127.0.0.1:5701:5701 -e HZ_CLUSTERNAME=source hazelcast/hazelcast:4.x.y
+----
+
+NOTE: For the `4.x.y` placeholder, you can use one of the released versions from Hazelcast 4 series, e.g., `4.2.7`.
+
+This member is started using an internal Docker IP and the port `5701` on that IP address, something like `172.12.0.1:5701`.
+Since the Docker container is in a separate network, it does not collide with the running processes on your
+local machine. For example, in this case, the member can run on `127.0.0.1:5701`, and our container can run on `172.12.0.1:5701`.
+
+However, since it runs in a virtual network, we need to make it accessible to your local processes because migration and target
+clusters, CLC, and DMT will be processes outside the Docker environment, directly running on your computer. The `-p` option in the Docker command allows you to 
+map a container's port to the host machine. The command assigns `5701` as the container's port, so `127.0.0.1:5701` becomes the Hazelcast member's address.
+`127.0.0.1` refers to your local machine's address. Make sure there is nothing else running on port `5701` on your local machine.
+
+NOTE: Instead of using Docker to start a member, you can also download one of the 4.x.y packages from https://hazelcast.com/open-source-projects/downloads/archives/#hazelcast-imdg[Hazelcast web site] and https://docs.hazelcast.com/imdg/4.2/getting-started[start a member]. Make sure to https://docs.hazelcast.com/imdg/4.2/clusters/creating-clusters[set the cluster's name] as `source`.
+
+=== Step 2. Create a Map and Put Data
+
+The `source.yaml`, which is contained in the DMT download package, is the Hazelcast CLC configuration file to access the source cluster to populate data.
+You can modify the `address` field in this file, if needed, in case you used a different port mapping than `5701` while starting the member, 
+or you started a cluster without using Docker and it started on a port other than `5701`.
+
+The content of `source.yaml`:
+
+[source,yaml]
+----
+cluster:
+  name: "source"
+  address: "127.0.0.1:5701"
+----
+
+Check that you can put an entry to the source cluster by running the following CLC command on your terminal.
+
+[source,shell]
+----
+clc -c source.yaml map --name my-map set key-1 value-1
+----
+
+??? _Any possible errors and ways of fixing them here_ ???
```

Comment:
If you get an error related to CLC not being able to connect the source cluster, check your port mapping is correct, the source cluster container is running and source.yaml configuration is correct.

Reply:
Can we add this sentence here?